src/sentry/workflow_engine/handlers/condition/event_frequency_handlers.py

```diff
@@ -89,7 +93,10 @@ class EventFrequencyPercentHandler(EventFrequencyConditionHandler, DataCondition
     }
 
     @staticmethod
-    def evaluate_value(value: list[int], comparison: Any) -> DataConditionResult:
-        if len(value) != 2:
+    def evaluate_value(value: WorkflowJob, comparison: Any) -> DataConditionResult:
+        if len(value.get("snuba_results", [])) != 2:
```

Comment:
is this a common scenario or a weird snuba blip?

Reply:
it's possible we don't have the snuba results when we are evaluating the triggers outside of delayed processing